CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+## How to Contribute
+
+We welcome contributions in the form of issues or pull requests!
+
+We want this to be a place where all are welcome to discuss and contribute, so please note that this project is released with a Contributor Code of Conduct. By participating in this project you agree to abide by its terms. Find the code of conduct in the ``CONDUCT.md`` file on GitHub.
+
+If you have a problem using Autofocus or see a possible improvement, open an issue in the GitHub issue tracker. Please be as specific as you can.
+
+If you see an open issue you'd like to be fixed, take a stab at it and open a PR!
+
+## Steps for Making a Pull Request
+
+1. Fork the project from GitHub.
+2. Clone the forked repo to your local disk. 
+
+```bash
+git clone https://github.com/<your_github_user_name>/autofocus.git
+```
+
+3. `cd` into the directory.
+4. Create a new branch.
+
+```bash
+git checkout -b my_awesome_new_feature
+```
+
+5. Make your changes.
+6. Complete all items in the [pull request checklist](https://github.com/uptake/autofocus/blob/master/.github/pull_request_template.md) that you feel comfortable with. Feel free to submit a PR with some loose ends, particularly if you are not sure about how to complete a checklist item.
+7. Submit your PR against the `dev` branch.
```

Comment:
Do you want to provide any guidance around expected response time? Like "maintenance of this project is a volunteer effort,  so please allow some time for a review. If you have not received on a review within `x` days, leave a comment on the pull request kindly requesting one".

I'm genuinely asking, not passive-aggressively saying you should. I haven't done that before on my projects but have started thinking about it!

Reply:
Good idea. Added.